packages/schematics/angular/application/index.ts

```diff
@@ -132,6 +136,9 @@ function addDependenciesToPackageJson(options: ApplicationOptions) {
     if (!options.skipInstall) {
       context.addTask(new NodePackageInstallTask());
     }
+    if (options.experimentalZoneless) {
+      removePackageJsonDependency(host, 'zone.js');
```

Comment:
We should avoid removing `zone.js` from the `package.json`, as this could break other projects within the monorepo that depend on it.